amgut/db/patches/0040.sql

```diff
@@ -0,0 +1,220 @@
+-- Sep. 9, 2016
+-- Plate Mapper test 01
+
+SET search_path TO ag;
```

Comment:
I am afraid that everything I created is in the ag schema. I thought the
barcode schema is full of obsolete items. Let me see if I can change that.
- Qiyun

On Fri, Sep 9, 2016 at 4:49 PM, Jose Navas <notifications@github.com> wrote:

> In amgut/db/patches/0040.sql
> <https://github.com/biocore/american-gut-web/pull/614#discussion_r78264540>
> :
>
> > @@ -0,0 +1,220 @@
> > +-- Sep. 9, 2016
> > +-- Plate Mapper test 01
> > +
> > +SET search_path TO ag;
>
> Ugh - which is the current schema in which this table leave?
> In the current layout there are 2 schemas: ag and barcodes. If the tables
> are in the barcodes schema it is ok to leave it there, but if the tables
> are in the ag schema that need to be changed (either to a new schema or to
> the barcode schema).
>
> Sorry about this I didn't think about it earlier.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/biocore/american-gut-web/pull/614/files/892ced8029b0f3d6df124390eee2ff67ed802eb9#r78264540>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEMVN5wn1_LFwNOa722UESI6aFw9c7hCks5qofCngaJpZM4J5WfP>
> .
>